### PR TITLE
Adds example for Rust API.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

## Summary
 - Adds dev container profile for cargo development
 - Adds cargo package that depends on maliput packages released on crates.io: https://crates.io/search/?q=maliput
 - Adds binary crate for consuming the maliput api.

## Test it
```
cargo run --bin road_geometry
```

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: maliput_cargo/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "maliput_cargo"
+version = "0.1.0"
+edition = "2021"
```
Comment (agalbachicar) [inline maliput_cargo/Cargo.toml:4]:
```suggestion
edition = "2024"
```

File: maliput_cargo/src/road_geometry.rs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    println!(
+        "\tLane ID: {}",
+        road_position_result.road_position.lane().id()
+    );
+    println!(
+        "\tLane Position: {}",
+        road_position_result.road_position.pos()
+    );
+    println!("");
+}
```
Comment (agalbachicar) [inline maliput_cargo/src/road_geometry.rs:90]: Shouldn't we return `0` here?
```suggestion
    println!("");
    
    return 0;
}
```

**Review by agalbachicar — CHANGES_REQUESTED** [review]
PTAL, minor comments.

File: maliput_cargo/Cargo.toml (hunk 1 above)
Comment (francocipollone) [reply maliput_cargo/Cargo.toml:4]: Check this: https://doc.rust-lang.org/cargo/reference/manifest.html#the-edition-field

File: maliput_cargo/src/road_geometry.rs (hunk 2 above)
Comment (francocipollone) [reply maliput_cargo/src/road_geometry.rs:90]: There is no need to return anything in the main function however we could return if we want a `Result<>` in that case we would do something like: 
```rust
fn main() -> Result<(), Box<dyn std::error::Error>> {
    if bla bla {
      Error{"}  
    }
    Ok(())

}
```
See https://doc.rust-lang.org/rust-by-example/error/result.html#using-result-in-main

However in this case we are not checking an error condition so I would leave it this way for the moment

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**